checkmate/__init__.py

```diff
@@ -0,0 +1,35 @@
+import tensorflow as tf
+from checkmate.tf2.util.load_keras_model import get_keras_model
+from checkmate.tf2.extraction import dfgraph_from_tf_function
+from checkmate.core.solvers.strategy_checkpoint_all import solve_checkpoint_all
+from checkmate.core.solvers.strategy_chen import solve_chen_greedy, solve_chen_sqrtn
+from checkmate.core.solvers.strategy_optimal_ilp import solve_ilp_gurobi
+from checkmate.tf2.execution import edit_graph
+
+print("Warning: importing checkmate currently turns off a single optimization in tensorflow on import")
```

Comment:
Swap this line to a call to `logging.warning`. Describe the pass as CSE, and capitalize Checkmate and TensorFlow